pkg/components/accesslog.go

```diff
@@ -68,6 +69,13 @@ func (c *loggingTransport) RoundTrip(r *http.Request) (*http.Response, error) {
 	if e == nil {
 		a.Status = resp.StatusCode
 		a.HTTPContentType = resp.Header.Get("Content-Type")
+		if resp.StatusCode < 200 || resp.StatusCode > 299 {
+			respData, err := io.ReadAll(resp.Body)
+			if err != nil {
+				runhttp.LoggerFromContext(r.Context()).Error(err)
+			}
+			a.Message = string(respData)
+		}
 	} else {
 		a.Status = transportd.ErrorToStatusCode(e)
```

Comment:
Suggestion: If we are logging non 2xx status code messages, it's probably good to keep consistent and do the same in this `else` block (as it will return a 5xx code). Instead of the message body we would likely just print the error.